src/CMakeLists.txt

```diff
@@ -395,22 +395,20 @@ ENDIF(OpenMVG_USE_OPENCV)
 # ==============================================================================
 # - optional, it allows to compile the vocabulary tree
 # ==============================================================================
-IF(OpenMVG_BUILD_VOCTREE)
-  OPTION(BOOST_NO_CXX11 "If Boost is not build with C++11 support this must be enabled to avoid symbol conflicts (SCOPED_ENUM)" OFF)
-  FIND_PACKAGE(Boost 1.53.0 QUIET COMPONENTS system filesystem program_options thread serialization)
-  IF(Boost_FOUND)
-     MESSAGE(STATUS "Boost ${Boost_LIB_VERSION} found")
-     if(BOOST_NO_CXX11)
-       add_definitions(-DBOOST_NO_CXX11_SCOPED_ENUMS)
-     endif(BOOST_NO_CXX11)
-     include_directories(${Boost_INCLUDE_DIRS})
-     link_directories(${Boost_LIBRARIES})
-     add_definitions(${Boost_DEFINITIONS})
-  ELSE()
-    MESSAGE(STATUS "Boost not found -- disabling the vocabulary tree")
-    UPDATE_CACHE_VARIABLE(OpenMVG_BUILD_VOCTREE OFF)
-  ENDIF(Boost_FOUND)
-ENDIF(OpenMVG_BUILD_VOCTREE)
+OPTION(BOOST_NO_CXX11 "If Boost is not build with C++11 support this must be enabled to avoid symbol conflicts (SCOPED_ENUM)" OFF)
```

Comment:
For the issue poparteu/openMVG#102 we need boost and it is not depending on the voctree.